cfgov/jinja2/v1/_includes/macros/video-player.html

```diff
@@ -55,7 +55,8 @@
     {% set button_pos   = options.button_pos or 'center' %}
     {% set is_flexible  = ( options.video.width or options.video.height ) is not defined %}
     <div class="video-player
-                video-player__{{ video_player }}"
+                video-player__{{ video_player }}
+                {{ 'featured-content-module_visual' if is_flexible == false }}"
```

Comment:
Would it be okay to do that and have video_player take in `options` and `additional_classes`? I couldn't figure out a way to have wagtail return a constant for is_fcm or additional classes so that content don't have to mess with that field.